service/src/test/java/uk/gov/hmcts/reform/fpl/service/additionalapplications/UploadAdditionalApplicationsServiceTest.java

```diff
@@ -162,7 +198,11 @@ void shouldOnlyConvertApplicationsWhenNotHMCTS() {
         AdditionalApplicationsBundle actual = underTest.buildAdditionalApplicationsBundle(caseData);
 
         assertThat(actual.getAuthor()).isEqualTo(USER_EMAIL);
-        assertThat(actual.getC2DocumentBundle().getDocument()).isEqualTo(DOCUMENT);
+        assertThat(actual.getC2DocumentBundle().getDocument()).isEqualTo(CONVERTED_DOCUMENT);
+        assertThat(actual.getC2DocumentBundle().getSupplementsBundle()).hasSize(1)
+            .first()
+            .extracting(actualSupplement -> actualSupplement.getValue().getDocument())
+            .isEqualTo(CONVERTED_SUPPLEMENT_DOCUMENT);
```

Comment:
these 2 tests have same content expect  the isHmcts value. It can be converted to parameterized test

Reply:
Pushed changes